### DISCUSSION TITLE
articles/david-reading-books-at-four-years-old

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#131] [Comments]
https://hutusi.com/articles/david-reading-books-at-four-years-old 

 任抛星汉归园圃，留取乾坤盛酒囊。

**Comment by flashclub** [comment]
好厉害啊，每天白天上班，晚上才有空陪宝宝，如何做到让他读这么多书的呢

**Comment by hutusi** [comment]
> @flashclub
> 好厉害啊，每天白天上班，晚上才有空陪宝宝，如何做到让他读这么多书的呢

坚持每天陪读，像 RAZ 有 App 可以每天打卡记录；周末多读一点；平时在家老人会带他读一些中文的绘本。还有就是听，牛听听、点读笔之类的，让他自己去探索。

**Comment by flashclub** [comment]
> > @flashclub
> > 好厉害啊，每天白天上班，晚上才有空陪宝宝，如何做到让他读这么多书的呢
> 
> 坚持每天陪读，像 RAZ 有 App 可以每天打卡记录；周末多读一点；平时在家老人会带他读一些中文的绘本。还有就是听，牛听听、点读笔之类的，让他自己去探索。

好的 谢谢解答

**Comment by marshhj** [comment]
佩服佩服

**Comment by hutusi** [comment]
@marshhj 看书太多对视力影响也大😪
